### PR TITLE
combined with everythin on latest layout and global themes, working scroll and bringing data from mock json server for upcoming cards

### PR DESCRIPTION
**Pull request by Liubzzz**

### CODE REVIEW

File: src/components/TabletApp/LeftSide/LeftSide.tsx [hunk 1]
```diff
@@ -22,95 +82,128 @@ const LeftSide = ({ roomName }: iLeftSide) => {
...
+                                {formattedDate}
+                            </Typography>
+                        </Box>
+                    </Box>
+                    <Box
+                        sx={{
+                            display: "flex",
+                            flexDirection: "column",
+                            alignItems: "center",
+                            color: "white",
```
Comment (cristiberzescu) [inline src/components/TabletApp/LeftSide/LeftSide.tsx:162]: same here

File: src/api/getRequests.ts [hunk 2]
```diff
@@ -11,6 +11,9 @@ const getRooms = async () => {
 const getRoomById = async (id: number) => {
     return await axios.get(`http://localhost:3001/rooms/${id}`);
 };
+const getMeetingsData= async()=>{
```
Comment (alexgaita) [inline src/api/getRequests.ts:14]: Missed a space here
```suggestion
const getMeetingsData = async()=>{
```

File: src/components/TabletApp/LeftSide/LeftSide.tsx [hunk 3]
```diff
@@ -1,13 +1,73 @@
...
-import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
+import "./UpcomingCardsScrollCSS.css";
+import Dayjs from "dayjs";
+import GlobalTheme from "../../../context/GlobalThemes";
+
 interface iLeftSide {
-    roomName: string;
+    name: string | undefined;
+    meetings:
+        | {
```
Comment (alexgaita) [inline src/components/TabletApp/LeftSide/LeftSide.tsx:12]: Create a new type for this meeting like Meeting where you specify the fields, don't inline them here

File: src/components/TabletApp/LeftSide/LeftSide.tsx [hunk 4]
```diff
@@ -1,13 +1,73 @@
...
+    },
+    {
+        start: "Wow",
+        end: "Tue Apr 03 2023 16:46:29 GMT+0300",
+        persons: ["ana beluci", "oana maria"],
+    },
+];
+
+const LeftSide = ({ name, meetings }: iLeftSide) => {
     let currDate = new Date();
```
Comment (alexgaita) [inline src/components/TabletApp/LeftSide/LeftSide.tsx:71]: use dayjs for dates

**Review by alexgaita — CHANGES_REQUESTED** [review]

**Review by CatalinTiu — CHANGES_REQUESTED** [review]
Check the file differences that must be resolved also! Everything else looks great!

**Review by CatalinTiu — APPROVED** [review]
Looks good to me

File: src/components/TabletApp/RightSide/MeetingInfo/MeetingInfo.tsx [hunk 5]
```diff
@@ -59,16 +38,6 @@ const MeetingInfo = ({
         return result.data;
     };
```
Comment (CatalinTiu) [inline src/components/TabletApp/RightSide/MeetingInfo/MeetingInfo.tsx:39]: Maybe use the getMeetingByID function from getRequests file or create it there and import it here

**Review by CatalinTiu — APPROVED** [review]

**Pull request merged**